### DISCUSSION TITLE
Paddle OCR 推理模型转ONNX，固定shape后，ONNX结果相差很大，不固定shape，结果与paddle推理模型保持一致，这个问题要怎么处理哇：

### DISCUSSION THREAD

**Discussion by xiaoxianyu12** [PaddlePaddle/PaddleOCR#13102] [Q&A]
Paddle OCR 推理模型转ONNX，固定shape后，ONNX结果相差很大，不固定shape，结果与paddle推理模型保持一致，这个问题要怎么处理哇：

尝试了2种方法：
①先将paddle推理模型的动态shape转为固定shape，再转成ONNX；
②先将paddle推理模型转为ONNX,此时ONNX模型为动态shape，再使用paddle2onnx.optimize将ONNX动态shape改为固定shape

这两种方法得到的固定shape的ONNX结果都和动态shape的结果有差异，请问这个怎么解决，怎么能够让Paddle OCR的模型转为固定shape的ONNX并且结果保持一致

**Comment by pcycccccc** [comment]
您好，我想请教一下，如何不固定shape进行转换呢？转换的指令是什么？
